src/app/components/activity/feed/devlog-post/media/item/item.ts

```diff
@@ -43,6 +43,17 @@ export default class AppActivityFeedDevlogPostMediaItem extends Vue {
 		return this.isActive;
 	}
 
+	get deviceMaxHeight() {
+		if (GJ_IS_SSR) {
+			return;
+		}
+
+		if (Screen.isMobile) {
+			return screen.height * 0.45;
```

Comment:
write this as `window.screen`. Since window is the "global" context that any variable attaches to, I try to reference it directly so that it's obvious when we're pulling from window, such as `window.location` and stuff

Reply:
Also, add a comment why we need this if!